str/trtools/stat_str_runner.py

```diff
@@ -23,42 +24,65 @@
 
 # inputs:
 # file-path
-@click.option('--file-path', help='gs://...')
+@click.option(
+    '--input-dir', help='gs://...file path if unsharded, input directory if sharded'
+)
 # caller
 @click.option(
     '--caller',
     help='gangstr or eh',
     type=click.Choice(['eh', 'gangstr'], case_sensitive=True),
 )
 @click.option(
-    '--job-storage', help='Storage of the Hail batch job eg 30G', default='50G'
+    '--job-storage', help='Storage of the Hail batch job eg 30G', default='16G'
 )
-@click.option('--job-memory', help='Memory of the Hail batch job eg 64G', default='32G')
+# sharded flag
+@click.option('--sharded', is_flag=True, help='Assume a sharded catalog was used')
+@click.option('--job-memory', help='Memory of the Hail batch job eg 64G', default='16G')
 @click.command()
 def main(
-    file_path, caller, job_storage, job_memory
+    input_dir, caller, job_storage, job_memory, sharded
 ):  # pylint: disable=missing-function-docstring
     # Initializing Batch
     b = get_batch()
-    vcf_input = b.read_input(file_path)
-    trtools_job = b.new_job(name=f'statSTR {caller}')
-
-    trtools_job.image(TRTOOLS_IMAGE)
-    trtools_job.storage(job_storage)
-    trtools_job.memory(job_memory)
-
-    trtools_job.declare_resource_group(ofile={'tab': '{root}.tab'})
-
-    trtools_job.command(
-        f"""
-        set -ex;
-        statSTR --vcf {vcf_input} --vcftype {caller} --out {trtools_job.ofile} --thresh --afreq --acount --hwep --het --entropy --mean --mode --var --numcalled
-
-        """
-    )
 
-    output_path_vcf = output_path(f'statSTR_samples_{caller}', 'analysis')
-    b.write_output(trtools_job.ofile, output_path_vcf)
+    if sharded:
+        input_vcf_dict = [
+            str(gspath) for gspath in to_path(f'{input_dir}').glob('*.vcf.gz')
+        ]
+
+    else:
+        input_vcf_dict = [input_dir]
```

Comment:
Should it be `[to_path(f'{input_dir}']` here as well?

Reply:
Yes, thanks!